src/main/java/com/microsoft/azure/vmagent/AzureVMCloud.java

```diff
@@ -1113,7 +1114,8 @@ public ListBoxModel doFillAzureCredentialsIdItems(@AncestorInPath Item owner) {
             return new StandardListBoxModel()
                     .includeEmptyValue()
                     .includeAs(ACL.SYSTEM, owner, AzureCredentials.class)
-                    .includeAs(ACL.SYSTEM, owner, AzureMsiCredentials.class);
+                    .includeAs(ACL.SYSTEM, owner, AzureMsiCredentials.class)
```

Comment:
No need to remove previous MSI ?

Reply:
For now, all the MSI related resources have been marked as deprecated. Maybe remove it in the next release since it is available temporarily.